__tests__/app.test.js

```diff
@@ -302,4 +302,21 @@ describe('/api/articles', () => {
             });
         })
     })
+})
+
+describe('/api/users', () => {
+    test('GET:200 sends an array of user objects to the client', () => {
+        return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then(({ body }) => {
+            body.users.forEach((user) => {
```

Comment:
When using a for each we must assert that there is a length as the for each can iterate over an empty array.